ir/instr.h

```diff
@@ -70,6 +70,15 @@ class Select final : public Instr {
   smt::expr getTypeConstraints(const Function &f) const override;
 };
 
+class Cttz final : public Instr {
```

Comment:
I didn't make it binop because second parameter to cttz is not really an operand unlike other binop instructions; it's just a flag. Though if you think making it binop is better idea, I can do that.

Reply:
Well, it's still a binary operation. Unless it is hard to fit in BinOp, I would prefer to move it there. We really don't want an explosion in the number of IR ops. Thanks!

Reply:
And since second operand in Cttz is a flag, Binop::getTypeConstraints() will not produce correct type constraints for Cttz unless I add custom code to allow exceptions for Cttz in getTypeConstraints(). Are you good with allowing such exceptions in Binop::getTypeConstraints()?

Reply:
That's fine. Once we see a few more we can think how to abstract it properly.

Reply:
Okay. It's parsed as binop now. Let me know if something doesn't look good to you.